cms/djangoapps/contentstore/views/tests/test_organizations.py

```diff
@@ -18,17 +18,26 @@ def setUp(self):
         self.staff = UserFactory(is_staff=True)
         self.client.login(username=self.staff.username, password='test')
         self.org_names_listing_url = reverse('organizations')
-        self.org_short_names = ["alphaX", "betaX", "orgX"]
-        for index, short_name in enumerate(self.org_short_names):
-            add_organization(organization_data={
-                'name': 'Test Organization %s' % index,
-                'short_name': short_name,
-                'description': 'Testing Organization %s Description' % index,
-            })
 
     def test_organization_list(self):
-        """Verify that the organization names list api returns list of organization short names."""
-        response = self.client.get(self.org_names_listing_url, HTTP_ACCEPT='application/json')
+        """
+        Verify that the organization names list api returns list of organization short names.
```

Comment:
We need to create a new test class `class TestEdlyOrganizationListing(TestCase):` with its own use case on `edly organization` based filtering.

Reply:
Done

Reply:
```
Verify that the organization names list API only returns Edly's enabled organizations.
```